src/main/java/movie/storage/service/mapper/OrderMapper.java

```diff
@@ -0,0 +1,29 @@
+package movie.storage.service.mapper;
+
+import java.util.ArrayList;
+import java.util.List;
+import movie.storage.model.Order;
+import movie.storage.model.Ticket;
+import movie.storage.model.dto.OrderResponseDto;
+import movie.storage.model.dto.TicketResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class OrderMapper {
+
+    public OrderResponseDto convertOrderToDto(Order order) {
+        OrderResponseDto orderResponseDto = new OrderResponseDto();
+        orderResponseDto.setId(order.getId());
+        orderResponseDto.setOrderDate(order.getOrderDate());
+        orderResponseDto.setUserEmail(order.getUser().getEmail());
+        List<TicketResponseDto> ticketResponseDtoList = new ArrayList<>();
+        for (Ticket ticket : order.getTickets()) {
+            TicketResponseDto ticketResponseDto = new TicketResponseDto();
```

Comment:
maybe it woould be better to extract ticket mapping to separate method?

Reply:
or even separate class

Reply:
Add fillTicketRespList method, I think there is too little functionality for creating a separate class.

Reply:
Okay, since you don't have controller and even service to work with tickets let's leave it here